packages/patrol_cli/lib/src/features/test/native_test_runner.dart

```diff
@@ -5,16 +5,24 @@ import 'package:patrol_cli/src/features/run_commons/test_runner.dart';
 
 typedef _Callback = Future<void> Function(String target, Device device);
 
-/// This class doesn't have a builder callback.
+/// Orchestrates running tests on devices.
 ///
-/// This means that it's not possible to split the build and run phases.
+/// It maps running T test targets on D devices, resulting in T * D test runs.
```

Comment:
```suggestion
/// It maps running T test targets on D devices.
```

It's not the final number, let's not give it then

Reply:
What do you mean?